dr/discovered/deployment-rbd/drpc.yaml

```diff
@@ -0,0 +1,26 @@
+---
+apiVersion: ramendr.openshift.io/v1alpha1
+kind: DRPlacementControl
+metadata:
+  name: drpc
+spec:
+  preferredCluster: dr1
+  drPolicyRef:
+    name: dr-policy
+  placementRef:
+    kind: Placement
+    # Match kustomization namePrefix
+    name: deployment-rbd-placement
+  pvcSelector:
+    matchLabels:
+      appname: busybox
+  protectedNamespaces:
+    - deployment-rbd
+  kubeObjectProtection:
+    captureInterval: 1m
```

Comment:
This matches the default dr-policy, but I'm not sure we need to use the same internal, resources are unlikely to change so capturing every minute may not be needed. Remove this to use the default?

Reply:
Removed